app/helpers/logger/external-logger-bigquery.js

```diff
@@ -32,8 +32,10 @@ const ExternalLoggerBigQuery = function (bqClient, displayId, cacheVersion, os)
     };
 
     return bqClient.insert("events", data, nowDate, getDateForTableName(nowDate))
-      .catch((e)=>{
-        log.file("Could not log to bq " + require("util").inspect(e, { depth: null }));
+      .catch((e)=> {
+        let message = "Could not log to bq " + require("util").inspect(e, { depth: null });
+        console.log(logDatetime + message);
```

Comment:
@stulees yea, first I thought it would be good to have it logging to the console. But I think you are right it is not needed. It has been removed. cheers